resources/songbooks.json

```diff
@@ -9,7 +9,7 @@
  ],
  "testing": [
    {
-     "name": "Songs For Our Generation",
+     "name": "Songs for Our Generation",
```

Comment:
Why lowercase "for" ?

Reply:
We can capitalize, they just need to match in both files so it can access the array

Reply:
Let's capitalize then.